scramv1-tool-conf.file

```diff
@@ -95,5 +95,16 @@ if [ -e $SCRAMV1_ROOT/bin/chktool ] ; then
   rm -f %i/errors.log
 fi
 
+
+py27List=`echo ${PYTHON27PATH} | tr ':' '\n'`
+
+mkdir -p %{i}/${PYTHON_LIB_SITE_PACKAGES}
+touch %{i}/${PYTHON_LIB_SITE_PACKAGES}/tool-deps.pth
```

Comment:
@davidlange6 , we also need to relocate this file in `%post` otherwise it will have build paths instead of install.